### PR TITLE
888 - External link support for tooltips

### PR DESCRIPTION
**Pull request by oleksanderkorn**
Issue https://github.com/Joystream/pioneer/issues/888

### CODE REVIEW

**Comment by oleksanderkorn** [comment]
@jodator please review

File: packages/ui/src/common/components/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -121,11 +121,16 @@ export const Tooltip = ({
               >
                 {tooltipTitle && <TooltipPopupTitle>{tooltipTitle}</TooltipPopupTitle>}
                 <TooltipText>{tooltipText}</TooltipText>
-                {tooltipLinkURL && (
+                {tooltipLinkURL && (tooltipLinkURL.startsWith('#') || tooltipLinkURL.startsWith('/')) ? (
```
Comment (jodator) [inline packages/ui/src/common/components/Tooltip/Tooltip.tsx:124]:
```suggestion
                {tooltipLinkURL && (isExternalLink(tooltipLinkUrl)) ? (
```

To make this condition a bit easier to read I would extract this to a helper function.

I think that there should be additional bracket around both components. ATM if `tooltipLinkURL` is falsy the `<TooltipExternalLink />`  will be rendered.

I think it should be something like:

```tsx
{tooltipLinkUrl && (
  isExternalLink(tooltipLinkURL) ? (
   <TooltipLink /> 
  ) : (
   <TooltipExternalLink /> 
  )
)} // <- two closing brackets
```

**Review by jodator — CHANGES_REQUESTED** [review]
Thank you for the PR 😃 

See my comment for small changes. Overall 👍

File: packages/ui/src/common/components/Tooltip/Tooltip.tsx (hunk 1 above)
Comment (oleksanderkorn) [reply packages/ui/src/common/components/Tooltip/Tooltip.tsx]: Makes sense, pushed the update.

**Comment by oleksanderkorn** [comment]
Updated the check as well:
```
const isExternalLink = () =>
    tooltipLinkURL && (tooltipLinkURL.startsWith('http://') || tooltipLinkURL.startsWith('https://'))
```

**Review by jodator — APPROVED** [review]

**Comment by jodator** [comment]
Thanks!

**Pull request merged**